config/v201/init_core.sql

```diff
@@ -17,6 +17,18 @@ CREATE TABLE IF NOT EXISTS AVAILABILITY(
     CHECK (EVSE_ID >= 0 AND CONNECTOR_ID >= 0 AND (EVSE_ID > 0 OR CONNECTOR_ID = 0))
 );
 
+-- Transactions --
+CREATE TABLE IF NOT EXISTS TRANSACTIONS (
+    TRANSACTION_ID TEXT PRIMARY KEY NOT NULL,
+    SEQ_NO INT NOT NULL,
+    MESSAGE_TYPE TEXT NOT NULL,
+    EVSE_ID INT NOT NULL,
+    CONNECTOR_ID INT NOT NULL,
+    ID_TOKEN TEXT NOT NULL,
+    TIME_START TEXT NOT NULL,
```

Comment:
The time can be an INT64 type. There is a automatic conversion from Datetime to int64 in `SQliteStatement` class.